### PR TITLE
Improve session list item accessibility

### PR DESCRIPTION
**Pull request by NUmeroAndDev**
## Issue
- close #ISSUE_NUMBER

## Overview (Required)
- Improved session list operability in talkback.

## Links
- 

## Screenshot
- Not changed design

Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt [hunk 1]
```diff
@@ -281,52 +285,72 @@ private fun SearchedItemListField(
                 SearchedHeader(day = dayToTimeTable)
             }
             items(sessions) { timetableItemWithFavorite ->
-                Box(
```
Comment (NUmeroAndDev) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt:284]: Removed unnecessary `Box` nesting.

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionListItem.kt [hunk 2]
```diff
@@ -41,6 +44,11 @@ import io.github.droidkaigi.confsched2022.model.TimetableItem
 import io.github.droidkaigi.confsched2022.model.TimetableItem.Session
 import io.github.droidkaigi.confsched2022.model.TimetableItemId
 import io.github.droidkaigi.confsched2022.model.secondLang
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toLocalDateTime
+
+internal val inFavoriteKey = SemanticsPropertyKey<Boolean>("inFavorite")
+internal var SemanticsPropertyReceiver.inFavorite by inFavoriteKey
```
Comment (NUmeroAndDev) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionListItem.kt:51]: ref: https://developer.android.com/jetpack/compose/testing?hl=en#custom-semantics-properties

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt [hunk 3]
```diff
@@ -281,52 +285,72 @@ private fun SearchedItemListField(
...
+                    }
+                )
+                Row(
                     modifier = Modifier
                         .fillMaxWidth()
                         .clickable { onItemClick(timetableItemWithFavorite.timetableItem.id) }
                         .padding(16.dp)
+                        .semantics(mergeDescendants = true) {
+                            customActions = listOf(
+                                CustomAccessibilityAction(
```
Comment (NUmeroAndDev) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt:302]: https://user-images.githubusercontent.com/13705006/191950425-9b7457a0-db53-4e77-a431-b25deef4b2bf.mp4

Favorite operation using Talkback has been changed to CustomAction.


ref: https://youtu.be/6LsaP6oKxMY?t=764

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionListItem.kt [hunk 4]
```diff
@@ -61,20 +69,34 @@ fun SessionListItem(
...
+                contentDescription = buildString {
+                    val startLocalDateTime = timetableItem.startsAt
+                        .toLocalDateTime(TimeZone.of("UTC+9"))
+                    val endLocalDateTime = timetableItem.endsAt
+                        .toLocalDateTime(TimeZone.of("UTC+9"))
+                    val startTime = startLocalDateTime.time.toString()
+                    val endTime = endLocalDateTime.time.toString()
+
+                    appendLine(timetableItem.title.currentLangTitle)
+                    append("$startTime ~ $endTime")
```
Comment (NUmeroAndDev) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionListItem.kt:82]: Added time information to reading session items in Talkback.

**Review by takahirom — APPROVED** [review]
I didn't know about CustomAccessibilityAction! Looks awesome!

**Pull request merged**